### PR TITLE
steam-acf: init at 0.1.0

### PR DESCRIPTION
**Pull request by chisui**
- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by legendofmiracles — APPROVED** [review]
LGTM

File: pkgs/tools/games/steam-acf/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    sha256 = "16q3md7cvdz37pqm1sda81rkjf249xbsrlpdl639r06p7f4nqlc2";
+  };
+
+  cargoSha256 = "0fzlvn0sl7613hpsb7ncykmcl53dgl8rzsg317nwkj2w679q4xq6";
+
+  # Requires network access
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Tool to convert Steam .acf files to JSON.";
```
Comment (figsoda) [inline pkgs/tools/games/steam-acf/default.nix:20]:
```suggestion
    description = "Tool to convert Steam .acf files to JSON";
```

File: pkgs/tools/games/steam-acf/default.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    homepage = "https://github.com/chisui/acf";
+    license = licenses.mit;
+    maintainers = with maintainers; [ chisui ];
```
Comment (figsoda) [inline pkgs/tools/games/steam-acf/default.nix:23]:
```suggestion
    maintainers = with maintainers; [ chisui ];
    mainProgram = "acf";
```

File: pkgs/tools/games/steam-acf/default.nix [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    repo = "acf";
+    rev = "v${version}";
+    sha256 = "16q3md7cvdz37pqm1sda81rkjf249xbsrlpdl639r06p7f4nqlc2";
+  };
+
+  cargoSha256 = "0fzlvn0sl7613hpsb7ncykmcl53dgl8rzsg317nwkj2w679q4xq6";
+
+  # Requires network access
+  doCheck = false;
+
```
Comment (figsoda) [inline pkgs/tools/games/steam-acf/default.nix:18]: builds without it, i don't think its needed
```suggestion
```

**Review by figsoda — CHANGES_REQUESTED** [review]

File: pkgs/tools/games/steam-acf/default.nix [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    rev = "v${version}";
+    sha256 = "16q3md7cvdz37pqm1sda81rkjf249xbsrlpdl639r06p7f4nqlc2";
+  };
+
+  cargoSha256 = "0fzlvn0sl7613hpsb7ncykmcl53dgl8rzsg317nwkj2w679q4xq6";
+
+  meta = with lib; {
+    description = "Tool to convert Steam .acf files to JSON";
+    homepage = "https://github.com/chisui/acf";
+    license = licenses.mit;
```
Comment (figsoda) [inline pkgs/tools/games/steam-acf/default.nix:19]:
```suggestion
    license = with licenses; [ mit /* or */ asl20 ];
```

**Review by figsoda — CHANGES_REQUESTED** [review]

File: pkgs/tools/games/steam-acf/default.nix (hunk 4 above)
Comment (chisui) [reply pkgs/tools/games/steam-acf/default.nix]: It would be nice if the meta block could be extracted from the `Cargo.toml` files.

**Comment by figsoda** [comment]
Result of `nixpkgs-review pr 135839` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>steam-acf</li>
  </ul>
</details>

**Review by figsoda — APPROVED** [review]

**Pull request merged**